manageiq-operator/pkg/helpers/miq-components/orchestrator.go

```diff
@@ -64,7 +64,7 @@ func NewOrchestratorDeployment(cr *miqv1alpha1.ManageIQ) (*appsv1.Deployment, er
 			},
 			corev1.EnvVar{
 				Name:  "DATABASE_REGION",
-				Value: "0",
+				Value: dbRegionCode(cr),
```

Comment:
This can be `spec.DatabaseRegion` as the initial value will be set already (see my other comment).

Reply:
Done